src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -114,7 +114,7 @@ public class DockerTemplate implements Describable<DockerTemplate> {
 
     /**
      * Default constructor; give an unusable instance.
-     * 
+     *
```

Comment:
Unnecessary change - please undo. See the [contributing guide's coding style section](https://github.com/jenkinsci/docker-plugin/blob/master/CONTRIBUTING.md#submitting-pull-requests) for details.